google/cloud/bigtable/bigtable_client_testing.bzl

```diff
@@ -1,5 +1,21 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 # DO NOT EDIT -- GENERATED BY CMake -- Change the CMakeLists.txt file if needed
-"""Automatically generated source lists for ${TARGET} - DO NOT EDIT."""
+
+"""Automatically generated source lists for bigtable_client_testing - DO NOT EDIT."""
+
```

Comment:
This blank line doesn't appear in other files... how is this possible?

Reply:
Three different types of files with different functions to generate them. Fixed.